packages/decoder/test/current/test/wire-test.js

```diff
@@ -1072,26 +1070,30 @@ describe("Over-the-wire decoding", function () {
     it("Decodes unambiguous custom errors", async function () {
       const { WireTest } = abstractions;
       const deployedContract = await WireTest.deployed();
-  
+
       const decoder = await Decoder.forContract(WireTest, {
         projectInfo: { artifacts: Contracts }
       });
-  
+
       let abiEntry = WireTest.abi.find(
         ({ type, name }) => type === "function" && name === "throwUnambiguous"
       );
       let selector = web3.eth.abi.encodeFunctionSignature(abiEntry);
-  
+
       //we need the raw return data, and contract.call() does not exist yet,
       //so we're going to have to use web3.eth.call()
-  
-      let data = await web3.eth.call({
-        to: deployedContract.address,
-        data: selector
-      });
+      let data;
+      try {
```

Comment:
Same question here as previous comment, naturally.  (And also later in this file but I won't mark them all.)

Reply:
@cds-amal same here :)